src/js/redux/reducers/aboutTheData/index.js

```diff
@@ -4,6 +4,13 @@
 */
 
 export const initialState = {
+    searchTerm: '',
+    searchResults: [
```

Comment:
This should be an object with properties.

```
searchResults: {
  submissions: [],
  publications: []
}
```

or to follow the current structure, two completely separate properties.